internal/nostr/nostr.go

```diff
@@ -669,6 +666,26 @@ func (svc *Service) processEvents(ctx context.Context, subscription *Subscriptio
 			"subscriptionId": subscription.ID,
 		}).Info("Received EOS")
 		receivedEOS = true
+
+		// Publish the NIP47 request once EOS is received
```

Comment:
I'm not sure waiting for EOS is right on re-connect, same with NWC :scream:

Reply:
Don't we always have to wait for EOS? It just gives us the historic events like the info event at first right (atleast for our relay it's only the info event) and we don't want to process that

> can't we do the same here rather than having a boolean?

Nice catch, thanks!